Misc/NEWS.d/next/Core_and_Builtins/2024-12-23-11-14-07.gh-issue-128192.02mEhD.rst

```diff
@@ -0,0 +1 @@
+Support digest authentication algorithm SHA-256 in :mod:`urllib.request`.
```

Comment:
```suggestion
Upgrade HTTP digest authentication algorithm for :mod:`urllib.request` by
supporting SHA-2/256 digest authentication as specified in :rfc:`7616`.
```

This also requires a What's New entry. However, we do not have any documentation concerning HTTP digest access authentication (we only have https://docs.python.org/3/library/urllib.request.html#abstractdigestauthhandler-objects but never do we mention which digests we are supporting). Nonetheless, maybe we can update the wording of that documented entry.

Reply:
thanks!